README.md

```diff
@@ -29,7 +31,17 @@ The sensor assumes the `Conn Id` is set to `fivetran`, however if you are managi
 
 ## Modules
 
-### [Fivetran Sensor Async](https://github.com/astronomer/airflow-provider-fivetran-async/tree/main/fivetran_provider_async/hooks/fivetran.py)
+### [Fivetran Operator Async](https://github.com/astronomer/airflow-provider-fivetran-async/tree/main/fivetran_provider_async/operators/fivetran.py)
+
+`FivetranOperatorAsync` submits a Fivetran sync job and monitors it on trigger for completion.
+It requires that you specify the `connector_id` of the sync job to start. You can find `connector_id` in the Settings page of the connector you configured in the [Fivetran dashboard](https://fivetran.com/dashboard/connectors).
+
+Import into your DAG via:
+```python
+from fivetran_provider_async.operators.fivetran import FivetranOperatorAsync
```

Comment:
Since this package is only about Fivetran, `.fivetran` after `from fivetran_provider_async.operators` seems redundant.

So instead, we should have the following:

```python
from fivetran_provider_async.operators import FivetranOperatorAsync
from fivetran_provider_async.sensors import FivetranSensorAsync
```

Reply:
Done in [a430049](https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/commits/a4300496e12aeba5d6673638067638d8ecdee9cc)